cli/pcluster/config/pcluster_config.py

```diff
@@ -69,6 +69,7 @@ def __init__(
         # "From Stack" initialization parameters:
         :param cluster_name: the cluster name associated to a running Stack,
         if specified the initialization will start from the running Stack
+        :param auto_refresh: if set, automatic variables
```

Comment:
let's complete the comment, something like:
```
If set, the parameters will be refreshed when there is a section update. 
The add/remove_section methods always call the config_updated method
and this one will call the refresh of the parameters only if __autorefresh is set to true.
```

Reply:
Done